code/modules/surgery/organs/heart.dm

```diff
@@ -81,9 +81,10 @@
 	var/blood_loss = 100 //600 blood is human default, so 5 failures (below 122 blood is where humans die because reasons?)
 
 	//How much to heal per pump, negative numbers would HURT the player
-	var/heal_brute = 0
-	var/heal_burn = 0
-	var/heal_oxy = 0
+	var/heal_brute = 5
+	var/heal_burn = 5
+	var/heal_oxy = 5
+	var/heal_tox = 5
```

Comment:
You're missing the point of the base level cursed heart if you're changing this.

The base level cursed heart is meant to be *punishment* for whoever receives it; subtypes are the ones that should heal/damage.